listener.go

```diff
@@ -102,20 +105,23 @@ func (l *listener) setupConn(sess quic.Session) (*conn, error) {
 	if err != nil {
 		return nil, err
 	}
-
 	remotePeerID, err := peer.IDFromPublicKey(remotePubKey)
 	if err != nil {
 		return nil, err
 	}
-
 	remoteMultiaddr, err := toQuicMultiaddr(sess.RemoteAddr())
 	if err != nil {
 		return nil, err
 	}
+	connScope, err := l.rcmgr.OpenConnection(network.DirInbound, false)
```

Comment:
Should we loop here, or return the error to the caller?

Reply:
let's bail.